nodes/BoolOp.py

```diff
@@ -4,18 +4,34 @@
 A boolean operation, 'or' or 'and'.
 
 Attributes:
-    - values  (Expr)
-        - The values involved.
-    - op      (Or | And)
+    - values  ([Expr])
+        - A list of values involved
+    - op      (str)
         - The operator, Or or And.
```

Comment:
While you're updating this, say `'or' or 'and'` (with the quotes!) because case does matter.

Reply:
I have made changes accordingly.